packages/react/src/Navbar.js

```diff
@@ -14,21 +14,34 @@ const propTypes = {
   children: PropTypes.node,
   portalConfig: PropTypes.shape({
     title: PropTypes.string.isRequired,
-    link: PropTypes.string.isRequired
+    link: PropTypes.string.isRequired,
   }).isRequired,
   loginConfig: PropTypes.shape({
     show: PropTypes.bool,
     loginLink: PropTypes.string.isRequired,
-    loginButtonMessage: PropTypes.string
+    loginButtonMessage: PropTypes.string,
   }),
   userDropdownConfig: PropTypes.shape({
     show: PropTypes.bool,
     logoutLink: PropTypes.string,
     additionalItems: PropTypes.node,
+    initials: PropTypes.string,
   }),
+  navLinks: PropTypes.arrayOf(
+    PropTypes.shape({
+      href: PropTypes.string.isRequired,
+      content: PropTypes.oneOf([PropTypes.string, PropTypes.node]).isRequired,
+    })
+  ),
 };
 
-function Navbar({ children, portalConfig, loginConfig, userDropdownConfig }) {
+function Navbar({
+  children,
+  portalConfig,
+  loginConfig,
+  userDropdownConfig,
+  navLinks,
+}) {
   let Link = "a";
```

Comment:
@ajswis @wayne-w-cloud do you guys know why `Link` is set as `a` instead of simply using `Link` component from `react-router-dom`?

Reply:
The app may not use react-router-dom. In which case, this would error because `Link` wouldn't be found.

Reply:
I think we can simply use the `react-router-dom`'s Link component on links that perform in-app routing, and for routing that goes outside the app (to DNAnexus) we can simply use the `<a>` tag?

Reply:
I think this component is dynamically assigning links based on component inputs, especially log-in / log-out.
The Link needs to be initiated and does not allow it to be assigned as an empty string. Assigning to 'a' is not the right solution in my opinion and we should import the Link from 'react-router-dom' and bind it to our components.

Reply:
oh I see. I believe we need to setup Routes in order to use the `Link` component from `react-router-dom`. I don't think Storybook has Routes setup. That might be the reason why it complained before